app/assets/stylesheets/components/_firm.scss

```diff
@@ -13,8 +13,10 @@
 }
 
 .firm__adviser-distance {
-  margin-bottom: $baseline-unit*2;
+  margin-left: -$baseline-unit/2;
```

Comment:
Oh, I didn't realise that but it makes total sense. So we should just use standard em/rem here?